apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/sharing-modal/index.tsx

```diff
@@ -30,6 +32,9 @@ const SharingModal: React.FC = () => {
 	const { launchpadScreenOption } = window?.launchpadOptions || {};
 	const { isDismissed, updateIsDismissed } = useSharingModalDismissed( isDismissedDefault );
 	const { __ } = useI18n();
+	const localeSlug = useLocale();
+	const isEnglish = localeSlug?.startsWith( 'en' );
+	const isPrivateBlog = window?.wpcomGutenberg?.blogPublic === '-1';
```

Comment:
I don't see how this will block on P2. If the P2 is public, I _think_ it will still show.. is that ok?

Reply:
Yeah, if p2 is public the share modal pops up. Think that's ok but just wanted to double check;

<img width="1182" alt="Screenshot 2023-05-18 at 17 10 47" src="https://github.com/Automattic/wp-calypso/assets/5560595/7cdb9a44-1f51-4e02-a977-2b95bd8eb261">

Reply:
> I suspect the primary issue is to not show it on private blogs and P2s since you'd never really want to promote private posts.
> > Yeah, if p2 is public the share modal pops up. Think that's ok

I agree here as well. 👍

Reply:
yep, I didn't even think about P2s being public but I agree that the thing we care about is private blogs. A public P2 would probably want to share and if not they can dismiss it.